src/styles/watch.scss

```diff
@@ -0,0 +1,63 @@
+.watch {
+  height: 500px;
```

Comment:
The height and width of the watch should be 80vmin as per the task description, not 500px. It should be changed to 'height: 80vmin;' and 'width: 80vmin;'.